src/test/java/com/epam/tc/hw2/ex1/Exercise1Test.java

```diff
@@ -0,0 +1,93 @@
+        softly.assertThat(driver.findElement(By.id("user-name")).getText()).isEqualTo("ROMAN IOVLEV");
+
+        /* Assert that there are 4 items on the header section are displayed and they have proper texts. */
+        softly.assertThat(driver.findElement(By.cssSelector("ul.m-l8>li>a[href='index.html']")).isDisplayed());
+        softly.assertThat(driver.findElement(By.cssSelector("ul.m-l8>li>a[href='contacts.html']")).isDisplayed());
+        softly.assertThat(driver.findElement(By.cssSelector("ul.m-l8>li>a.dropdown-toggle")).isDisplayed());
+        softly.assertThat(driver.findElement(By.cssSelector("ul.m-l8>li>a[href='metals-colors.html']")).isDisplayed());
+
+        softly.assertThat(driver.findElement(By.cssSelector("ul.m-l8>li>a[href='index.html']")).getText()).isEqualTo("HOME");
+        softly.assertThat(driver.findElement(By.cssSelector("ul.m-l8>li>a[href='contacts.html']")).getText()).isEqualTo("CONTACT FORM");
+        softly.assertThat(driver.findElement(By.cssSelector("ul.m-l8>li>a.dropdown-toggle"))
+                .getText()).isEqualTo("SERVICE");
+        softly.assertThat(driver.findElement(By.cssSelector("ul.m-l8>li>a[href='metals-colors.html']")).getText()).isEqualTo("METALS & COLORS");
+        softly.assertAll();
+
+
+        /* Assert that there are 4 images on the Index Page and they are displayed.*/
+        softly.assertThat(driver.findElement(By.className("icon-practise")).isDisplayed());
+        softly.assertThat(driver.findElement(By.className("icon-custom")).isDisplayed());
+        softly.assertThat(driver.findElement(By.className("icon-multi")).isDisplayed());
+        softly.assertThat(driver.findElement(By.className("icon-base")).isDisplayed());
+        softly.assertAll();
+
+        /* Assert that there are 4 texts on the Index Page under icons and they have proper text. */
+        softly.assertThat(driver.findElement(By.xpath("//div[normalize-space(span) = "
+                + "'To include good practicesand ideas from successfulEPAM project']")).isDisplayed());
+        softly.assertThat(driver.findElement(By.xpath("//div[normalize-space(span) = "
+                + "'To be flexible andcustomizable']")).isDisplayed());
+        softly.assertThat(driver.findElement(By.xpath("//span[(text()='To be multiplatform ')]")).isDisplayed());
+        softly.assertThat(driver.findElement(By.xpath("//div[normalize-space(span) = 'Already "
+                + "have good base(about 20 internal andsome external projects),wish to get more…']")).isDisplayed());
+        softly.assertAll();
+
+        /* Assert that there is the iframe with “Frame Button” exist. */
+        boolean frameIsDisplayed = driver.findElement(By.id("frame")).isDisplayed();
+        softly.assertThat(frameIsDisplayed).isTrue();
+
+        /* Switch to the iframe and check that there is “Frame Button” in the iframe. */
+        driver.switchTo().frame("frame");
+        WebElement frameButton = driver.findElement(By.id("frame-button"));
```

Comment:
The same comment here. There is no need to use this extra boolean variable here. Please, simplify and use as() method to provide some description.